src/state/routing/types.ts

```diff
@@ -68,20 +67,20 @@ export interface GetQuoteResult {
   routeString: string
 }
 
-export class InterfaceTrade<
+export class ClassicTrade<
   TInput extends Currency,
```

Comment:
nit: Can someone create another type based on ClassicTrade and supplement different values here? Do we do that or plan to do it in the future?

If not, I would vote to make it a simple ClassicTrace class that extends `Trade<Currency, Currency, TradeType>`